app/controllers/alchemy/admin/pages_controller.rb

```diff
@@ -83,6 +83,9 @@ def edit
         if page_is_locked?
           flash[:notice] = Alchemy.t('This page is locked', name: @page.locker_name)
           redirect_to admin_pages_path
+        elsif !@page.editable_by?(current_alchemy_user)
+          flash[:notice] = Alchemy.t('Your user role does not allow you to edit this page.')
+          redirect_to admin_pages_path
```

Comment:
Ah, ok we don't use the automagically authorize! of cancan in this controller. So, this can be left in. Sorry for the confusion @Brantron 

But: Please change the flash type to warning